src/components/tasks/task-dialog.tsx

```diff
@@ -430,10 +568,11 @@ const TaskDialog = ({ projectId, milestoneId, open, task, onClose }: Props) => {
             variant="contained"
             color="primary"
             size="large"
+            sx={{ minHeight: 50, marginTop: 2, marginBottom: 2}}
             disabled={isDisabled}
           >
-            <AddIcon />
-            {task ? "Update task" : t("newMilestoneTaskDialog.createButton")}
+            {!task && <AddIcon />}
```

Comment:
Why is this conditional?

Reply:
AddIcon is visible next to a button label only if we are creating a new task. It is not shown if we are updating an existing task